app/views/podcasts/show.rss.builder

```diff
@@ -51,7 +51,7 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
     xml.itunes :author, @podcast.author_name unless @podcast.author_name.blank?
     xml.itunes :type, @podcast.itunes_type unless @podcast.itunes_type.blank?
 
-    @podcast.itunes_categories[0, 3].each do |cat|
+    @itunes_categories[0, 3].each do |cat|
```

Comment:
out of curiosity what is the value of those indexes (`[0, 3]`)?

Reply:
That's just a less-common syntax for getting the first 3 items from an array.  (Not sure why we grab the first 3?  I think Apple generally only parses 1.  But left this logic intact).